admin/spinw_on.m.template

```diff
@@ -1,75 +0,0 @@
-function path=spinw_on(non_default_path)
```

Comment:
This file should not be removed. It's used on IDAaaS to add SpinW to the path when you do `horace_on`.

Reply:
Done that.